### PR TITLE
MTA-546: Migration Waves

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/MTA-546

MTA 6.2.0

Preview:
https://deploy-preview-711--windup-documentation.netlify.app/docs/web-console-guide/master/#mta-web-creating-migration-waves_user-interface-guide
https://deploy-preview-711--windup-documentation.netlify.app/docs/web-console-guide/master/#mta-web-creating-jira-issues-for-migration-wave_user-interface-guide
https://deploy-preview-711--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-web-application-attributes_user-interface-guide

### CODE REVIEW

File: docs/topics/mta-web-creating-migration-waves.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-migration-waves_{context}"]
+= Creating migration waves
+
+A migration wave is a tool for creating Jira issues for a group of applications that are to be migrated at the same time. For each application assigned to a migration wave, {ProductShortName} opens a separate Jira issue with an automatically generated title.
```
Comment (gildub) [inline docs/topics/mta-web-creating-migration-waves.adoc:9]: The primary goal of a migration wave is to offer the user a way to group applications to be migrated on a given schedule. Each application of a migration wave can be exported to external issue management system (only Jira for now).

File: docs/topics/mta-web-creating-migration-waves.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.Procedure
+
+. In *Migration* view, click *Migration waves*.
+. Click *Create new*.
++
+The *New migration wave* window opens.
+. Enter the following information or select from drop-down lists:
+* Name
```
Comment (gildub) [inline docs/topics/mta-web-creating-migration-waves.adoc:18]: Name is optional (in which case the start and end dates are used to distinguish migration waves.

**Review by abrugaro — APPROVED** [review]
LGTM taking into account @gildub 's comments

File: docs/topics/mta-web-creating-migration-waves.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-creating-migration-waves.adoc:9]: Corrected

File: docs/topics/mta-web-creating-migration-waves.adoc (hunk 2 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-creating-migration-waves.adoc:18]: Corrected

**Comment by sbeskin-redhat** [comment]
@gildub Thank you for the review! Please take a look now.

File: docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-jira-issues-for-migration-wave_{context}"]
+= Creating Jira issues for a migration wave
+
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc:8]: Can you create more than one Jira issue for a single migration wave? If yes, then the title is great. If not, I'd suggest either "Creating Jira issues for migration waves" or "Creating a migration issue for a migration wave". If "not," then you'll also have to adjust the subject-agreement in the passage,  depending on which title option you use. .

File: docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+You can use a migration wave for creating Jira issues automatically for each application assigned to it.
+
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc:10]: Suggestion: Replace "it" with "the migration wave."

File: docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+.Procedure
+
+. In *Migration* view, click *Migration waves*.
+. From the *Options* menu (three dots) of the migration wave for which you want to create Jira issues, select *Export to Issue Manager*.
++
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc:15]: We usually write this as "Click the Options menu {kebab} to the right of ... " with Options not bolded (unless it is actually called "Options" in the UI.

{kebab} is the attribute for the three dots in the windup-documentation repo -- it renders as the three dots. 

Please use the attribute and consider the rewrite I suggested for the sake of consistency in the repo.

File: docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+The *Export to Issue Manager* window opens.
+. Select the instance type (Jira Cloud or Jira Server/Datacenter).
+. Select the instance, project and issue type from the drop-down lists.
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc:18]: "...instance, project, and issue type from the lists." More than one list, correct? I can't see the UI, so pardon "obvious" questions.  IBM says not to use "drop-down" or any other modifiers for lists, unless more than one type of list is present at the part of the UI (I paraphrase).

File: docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+. Click *Export*.
++
+The status of the migration wave on the *Migration waves* page changes to `Issues Created`. To see the status of each individual application of a migration wave, click the *Status* column.
++
+A separate Jira issue has been created for each application associated with the migration wave. The following fields of each issue are filled in automatically:
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc:23]: Suggestion: Change "has been" to "is." The present simple is accurate here.

File: docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+* Title: `Migrate <app name>`
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc:25]: Looking at the guide, we use "application" much more often than "app," but you know the context better than I do.

File: docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+* Reporter: Username of the token owner
+* Description: `Created by Konveyor`
+
+[NOTE]
+====
+Once the Jira issues have been created after you exported a migration wave to the Issue Manager, you can no longer change them from within the {ProductShortName} {WebName}. Even if you delete the migration wave, the Jira issues remain.
+====
+
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc:33]: In the first sentence, is "after you exported a migration wave to the Issue Manager" relevant? That is, are you saying the same thing as "Once the Jira issues have been created,..." ? If the phrase is not relevant, I suggest deleting it. If it is relevant, I suggest starting the sentence with it "After you export a migration wave to the Issue Manager, you can no longer change any Jira issues related to the migration wave from inside the {ProductShortName} {WebName}."

The phrasing of the note leaves open the possibility that you can change the Jira issues from outside the UI. If that is so, consider specifying how to. If it's not so, consider de;etig "from within..."

File: docs/topics/mta-web-creating-migration-waves.adoc [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+. Enter the following information or select from drop-down lists:
+* Name (optional; if not given, you can use the start and end dates to identify migration waves)
+* Potential start date; it must be later than the current date
+* Potential end date; it must be later than the start date
+* Stakeholders (optional)
+* Stakeholder groups (optional)
+. Click *Create*.
++
+The new migration wave appears in the list of existing migration waves.
+. To assign applications to a migration wave, click its *Options* menu (three dots) and select *Manage applications* from the menu.
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-migration-waves.adoc:28]: See comment about {kebab} above.

**Comment by RichardHoch** [comment]
Good, solid writing. A few small comments, but otherwise LGTM.

**Review by kpunwatk — APPROVED** [review]
Reviewed and LGTM :+1:

**Review by rromannissen — APPROVED** [review]
LGTM!

**Pull request merged**